ansible/roles/common/templates/conf/output/00-local.conf.j2

```diff
@@ -15,6 +15,8 @@
        logstash_format true
        logstash_prefix {{ kibana_log_prefix }}
        flush_interval 15s
+       reload_on_failure true
```

Comment:
My understanding of these two settings is that they merely pull a new host list (usually that involves looking up a DNS record, which doesn't apply in our case.) What about the `reconnect_on_error` setting that defaults to `false`? Maybe that is more appropriate.

`reload_connections` should be defaulting to `true` so I don't think that line will do anything.

Reply:
We can try that. I'm a bit concerned about it reconnecting each time we get this horizon error about an empty first line.

Reply:
I understand the docs saying that it only applies to transport errors:

> Indicates that the plugin should reset connection on any error (reconnect on next send).

But, good point, we should look out for that.